src/server/debugcmd.cc

```diff
@@ -104,6 +104,8 @@ void DebugCmd::Run(CmdArgList args) {
         "      existing RDB file.",
         "REPLICA PAUSE/RESUME",
         "    Stops replica from reconnecting to master, or resumes",
+        "REPLICA OFFSET",
```

Comment:
mention that you send a sync id together with the array of offsets for each replication flow.